### PR TITLE
Create station schema

### PR DESCRIPTION
**Pull request by SvetaMemphis**
Add choose schema to create station form, getStareted excluded.

### CODE REVIEW

File: ui_src/src/components/createStationForm/style.scss [hunk 1]
```diff
@@ -23,8 +23,9 @@
     .storage-replicas-container {
         display: flex;
         width: 100%;
+        justify-content: space-between;
         .storage-container {
-            justify-content: space-between;
+            // justify-content: space-between;
```
Comment (avrhamNeeman) [inline ui_src/src/components/createStationForm/style.scss:28]: remove

Comment (SvetaMemphis) [reply ui_src/src/components/createStationForm/style.scss]: Done

File: ui_src/src/components/createStationForm/index.js [hunk 2]
```diff
@@ -331,6 +345,29 @@ const CreateStationForm = ({ createStationFormRef, getStartedStateRef, finishUpd
...
+                        <SelectSchema
+                            height="40px"
+                            width="450px"
+                            placeholder={creationForm.schemaValue || 'Select schema'}
+                            value={creationForm.schemaValue}
+                            colorType="navy"
+                            radiusType="semi-round"
+                            options={schemas}
+                            onChange={(e) => creationForm.setFieldsValue({ schemaValue: e })}
+                            // boxShadowsType="gray"
```
Comment (avrhamNeeman) [inline ui_src/src/components/createStationForm/index.js:364]: You can only receive these props: {options, onChange, value, placeholder}. Please delete unnecessary fields.

File: ui_src/src/components/selectSchema/style.scss [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            width: 67px;
+            height: 22px;
+            margin-right: 10px;
+            text-align: center;
+            p {
+                margin-right: 0px;
+                font-family: 'InterMedium';
+                font-size: 12px;
+            }
+        }
```
Comment (avrhamNeeman) [inline ui_src/src/components/selectSchema/style.scss:69]: ??

Comment (avrhamNeeman) [reply ui_src/src/components/selectSchema/style.scss:69]: Delete all unnecessary classes.

File: ui_src/src/components/createStationForm/index.js (hunk 2 above)
Comment (SvetaMemphis) [reply ui_src/src/components/createStationForm/index.js:364]: Done

File: ui_src/src/components/selectSchema/style.scss (hunk 3 above)
Comment (SvetaMemphis) [reply ui_src/src/components/selectSchema/style.scss:69]: Done

**Pull request merged**